config/config.go

```diff
@@ -1285,33 +1285,68 @@ func (c *Config) addInput(name string, table *ast.Table) error {
 	c.setLocalMissingTomlFieldTracker(missCount)
 	defer c.resetMissingTomlFieldTracker()
 
+	// Handle removed, deprecated plugins
+	if di, deprecated := inputs.Deprecations[name]; deprecated {
```

Comment:
deprecation was checked only if `Inputs[name]` wasn't found, but  now it is executed regardless unconditionally, hope it's OK